Makefile

```diff
@@ -39,18 +39,20 @@ EXTERNAL_SRCS =
 ifeq ($(CRYPTO), openssl)
   _LDFLAGS += -lcrypto
   _CFLAGS += -DSECVAR_CRYPTO_OPENSSL
-  EXTERNAL_SRCS += external/skiboot/libstb/secvar/crypto/crypto-openssl.c
+  EXTERNAL_SRCS += external/libstb-secvar/src/crypto_openssl.c
```

Comment:
I think this should be included by default now as part of `libstb-secvar-openssl.a`, so we might not need to include external `*.c` files at all for crypto once all are ported over.